tests/test_secrets.py

```diff
@@ -337,3 +337,46 @@ def test_9_2_GETKey(self):
         self.assertEqual(rep['output'],
                          {"type": "simple", "value":
                           b64encode(b'1234').decode('utf-8')})
+
+    def test_10_CREATEsubcont(self):
+        # Create a container
+        req = {'remote_user': 'test',
+               'trail': ['test', 'container_a', '']}
+        rep = {'headers': {}}
+        self.POST(req, rep)
+        self.assertEqual(rep['code'], 201)
+
+        # Create a subcontainer
+        req = {'remote_user': 'test',
+               'trail': ['test', 'container_a', 'sub', '']}
+        rep = {'headers': {}}
+        self.POST(req, rep)
+        self.assertEqual(rep['code'], 201)
+
+        # Create a secret over the parent container
+        req = {'headers': {'Content-Type': 'application/json'},
+               'remote_user': 'test',
+               'trail': ['test', 'container_a', 'key2'],
+               'body': '{"type":"simple","value":"1234"}'.encode('utf-8')}
+        rep = {}
+        self.PUT(req, rep)
+
+        req = {'remote_user': 'test',
+               'trail': ['test', 'container_a']}
+        rep = {'headers': {}}
+        self.GET(req, rep)
+        # self.assertEqual(rep['output'], ['sub', 'key2'])
```

Comment:
This should be ['sub/', 'key2'], we currently return ['sub', key2'] which does not allow us to undertand the difference between key and container.